src/floating-focus.js

```diff
@@ -150,17 +150,29 @@ export default class FloatingFocus {
 		this.target.classList.remove('focus');
 	}
 
+	addPixels(pixels1, pixels2) {
+		return `${parseFloat(pixels1) + parseFloat(pixels2)}px`;
+	}
```

Comment:
Or maybe better to make into a static method / even a util method outside of the class?
(The reason I did it like this is that e.g. `repositionElement()` also does nothing with `this` and yet still is a class method)